poc/vidpf.py

```diff
@@ -81,6 +81,10 @@ def root(cls, seed: bytes, ctrl: bool):
         return cls(seed, ctrl, [], bytes())
 
 
+def identity_transform(w: list[F], _level: int) -> list[F]:
```

Comment:
Oh, that's nice. Initially I was about to comment why we need to abstract the truncate inside the transform in mastic, but I see why now, nice.